hw/top_earlgrey/syn/asic.constraints.sdc

```diff
@@ -215,30 +215,109 @@ set_output_delay ${SPI_HOST_OUT_DEL} [get_ports SPI_HOST_D1]   -clock SPI_HOST_C
 set_output_delay ${SPI_HOST_OUT_DEL} [get_ports SPI_HOST_D2]   -clock SPI_HOST_CLK
 set_output_delay ${SPI_HOST_OUT_DEL} [get_ports SPI_HOST_D3]   -clock SPI_HOST_CLK
 
+#####################################
+# SPI DEV clock Passthru Operation  #
+#####################################
+# Passthrough target freq is 33MHz. Using 40MHz to over-constrain
```

Comment:
i'll link a diagram here eventually to help clarify what i'm doing.

Reply:
Nice. Thanks for adding these constraints!